src/Configuration.jl

```diff
@@ -280,108 +281,66 @@ Collection of all settings that configure a Pluto session.
-        workspace_use_distributed::Bool = WORKSPACE_USE_DISTRIBUTED_DEFAULT,
-        workspace_use_distributed_stdlib::Union{Bool,Nothing} = WORKSPACE_USE_DISTRIBUTED_STDLIB_DEFAULT,
-        lazy_workspace_creation::Bool = LAZY_WORKSPACE_CREATION_DEFAULT,
-        capture_stdout::Bool = CAPTURE_STDOUT_DEFAULT,
-        workspace_custom_startup_expr::Union{Nothing,String} = WORKSPACE_CUSTOM_STARTUP_EXPR_DEFAULT,
-
-        compile::Union{Nothing,String} = COMPILE_DEFAULT,
-        pkgimages::Union{Nothing,String} = PKGIMAGES_DEFAULT,
-        compiled_modules::Union{Nothing,String} = COMPILED_MODULES_DEFAULT,
-        sysimage::Union{Nothing,String} = SYSIMAGE_DEFAULT,
-        sysimage_native_code::Union{Nothing,String} = SYSIMAGE_NATIVE_CODE_DEFAULT,
-        banner::Union{Nothing,String} = BANNER_DEFAULT,
-        depwarn::Union{Nothing,String} = DEPWARN_DEFAULT,
-        optimize::Union{Nothing,Int} = OPTIMIZE_DEFAULT,
-        min_optlevel::Union{Nothing,Int} = MIN_OPTLEVEL_DEFAULT,
-        inline::Union{Nothing,String} = INLINE_DEFAULT,
-        check_bounds::Union{Nothing,String} = CHECK_BOUNDS_DEFAULT,
-        math_mode::Union{Nothing,String} = MATH_MODE_DEFAULT,
-        heap_size_hint::Union{Nothing,String} = HEAP_SIZE_HINT_DEFAULT,
-        startup_file::Union{Nothing,String} = STARTUP_FILE_DEFAULT,
-        history_file::Union{Nothing,String} = HISTORY_FILE_DEFAULT,
-        threads::Union{Nothing,String,Int} = default_number_of_threads(),
-    )
-    server = ServerOptions(;
-        root_url,
-        base_url,
-        host,
-        port,
-        port_hint,
-        launch_browser,
-        dismiss_update_notification,
-        dismiss_motivational_quotes,
-        show_file_system,
-        notebook_path_suggestion,
-        disable_writing_notebook_files,
-        auto_reload_from_file,
-        auto_reload_from_file_cooldown,
-        auto_reload_from_file_ignore_pkg,
-        notebook,
-        simulated_lag,
-        simulated_pkg_lag,
-        injected_javascript_data_url,
-        on_event,
-    )
-    security = SecurityOptions(;
-        require_secret_for_open_links,
-        require_secret_for_access,
-        warn_about_untrusted_code,
-    )
-    evaluation = EvaluationOptions(;
-        run_notebook_on_load,
-        workspace_use_distributed,
-        workspace_use_distributed_stdlib,
-        lazy_workspace_creation,
-        capture_stdout,
-        workspace_custom_startup_expr,
-    )
-    compiler = CompilerOptions(;
-        compile,
-        pkgimages,
-        compiled_modules,
-        sysimage,
-        sysimage_native_code,
-        banner,
-        depwarn,
-        optimize,
-        min_optlevel,
-        inline,
-        check_bounds,
-        math_mode,
-        heap_size_hint,
-        startup_file,
-        history_file,
-        threads,
-    )
-    return Options(; server, security, evaluation, compiler)
+@option struct PlutoOptions
+    Pluto:: Options = Options
+end
```

Comment:
This struct is specifically introduced so we stay compatible with PlutoDeploymentSettings 😅